### ISSUE TITLE
Tutorials-Show doc browser

### ISSUE THREAD

**Issue by ForebearingAbode** [sofa-framework/sofa#5058]
Hi I downloaded the windows .exe file to install SOFA, I attempted to go through the tutorials and eventually learned that I have to click show doc browser for the walk throughs. But I get the warning: "[WARNING] [RealGUI] Doc browser has been disabled because Qt5WebEngine is not available"

During installation I didn't see an option to activate this. I downloaded [qt-unified-windows-x64-online.exe](https://download.qt.io/official_releases/online_installers/qt-unified-windows-x64-online.exe), to no avail. 

I saw a similar post, but for someone who build it on linux using cmake. I tried following the steps for that but couldn't complete when I tried to run a command given that did nothing. 

I want that when I click show doc browser in SOFA it pops open that little window that I saw in a video tutorial. How do I get this?

**Comment by alxbilger** [comment]
If you downloaded the binaries, I am afraid that the doc browser is not shipped with them, and there is nothing you can do to get them. I am not sure if it is intended. I let @bakpaul know of this.

As an alternative, you could open the `.html` file (just next to the simulation scene file) in your favorite web browser. Just note that links to run simulations won't work.

If you really want the Doc Browser, you will have to compile SOFA by yourself.

**Comment by ForebearingAbode** [comment]
I wish I had an example of the tutorials provided in the doc browser to know if it is worth it or not. I just saw mention of it. I'd hate to put in the time figuring out how to compile it just for it to be another bust if it's no better than reading through the documentation while looking at the scn files.

**Comment by ForebearingAbode** [comment]
Even better would be if they were found online so this bug can't get in the way

**Comment by alxbilger** [comment]
If you could just open the html files with your browser as I already mentioned, it would save us time both.

**Comment by ForebearingAbode** [comment]
Yes I should've mentioned, that was the first thing I did while looking through the files and it just takes me to a picture of the model and what looks links that don't work.
![html files](https://github.com/user-attachments/assets/3f2e8b5c-d976-48f4-9e4a-1c7dc918843d)

**Comment by ForebearingAbode** [comment]
I just don't want to be missing out on a useful resource if it can be helped; I love working through the tutorials for new software, but always end up reading through the documentation anyway. So long as I can get most everything I'd need to know from that I should be fine if there's nothing that can be done about the html file problem (I had just assumed they didn't work because of being outdated but if you can tell what the above issue is that'd be much appreciated).
